5.0 build fails on Mac

Using MacOs 11.5.2 

```
git clone https://github.com/hazelcast/hazelcast ; cd hazelcast ; mvn install
```

gives

```
[INFO] Reactor Summary for Hazelcast Root 5.0-SNAPSHOT:
[INFO] 
[INFO] Hazelcast Root ..................................... SUCCESS [  2.451 s]
[INFO] hazelcast .......................................... SUCCESS [  02:20 h]
[INFO] hazelcast-spring ................................... SUCCESS [  8.080 s]
[INFO] hazelcast-spring-tests ............................. FAILURE [04:48 min]
[INFO] hazelcast-build-utils .............................. SUCCESS [ 14.620 s]
[INFO] hazelcast-jet-extensions ........................... SUCCESS [  0.078 s]
[INFO] hazelcast-jet-kafka ................................ SUCCESS [02:36 min]
[INFO] hazelcast-jet-avro ................................. SUCCESS [ 14.037 s]
[INFO] hazelcast-jet-csv .................................. SUCCESS [  5.333 s]
[INFO] hazelcast-jet-hadoop-core .......................... SUCCESS [01:04 min]
[INFO] hazelcast-sql ...................................... SUCCESS [11:02 min]
[INFO] modulepath-tests ................................... SUCCESS [ 12.590 s]
[INFO] hazelcast-jet-cdc-debezium ......................... SUCCESS [ 43.593 s]
[INFO] hazelcast-jet-cdc-mysql ............................ SUCCESS [01:23 min]
[INFO] hazelcast-jet-cdc-postgres ......................... SUCCESS [01:13 min]
[INFO] hazelcast-jet-elasticsearch-5 ...................... SUCCESS [ 32.220 s]
[INFO] hazelcast-jet-elasticsearch-6 ...................... SUCCESS [ 30.369 s]
[INFO] hazelcast-jet-elasticsearch-7 ...................... SUCCESS [03:49 min]
[INFO] hazelcast-jet-hadoop-dist .......................... SUCCESS [  0.152 s]
[INFO] hazelcast-jet-files-azure .......................... SUCCESS [01:08 min]
[INFO] hazelcast-jet-files-gcs ............................ SUCCESS [01:11 min]
[INFO] hazelcast-jet-files-s3 ............................. SUCCESS [03:33 min]
[INFO] hazelcast-jet-hadoop ............................... SUCCESS [  0.578 s]
[INFO] hazelcast-jet-hadoop-all ........................... SUCCESS [ 55.413 s]
[INFO] hazelcast-3-connector-root ......................... SUCCESS [  0.100 s]
[INFO] hazelcast-3-connector-interface .................... SUCCESS [  1.581 s]
[INFO] hazelcast-3-connector-impl ......................... SUCCESS [  2.298 s]
[INFO] hazelcast-3-connector-common ....................... SUCCESS [ 45.251 s]
[INFO] hazelcast-jet-kinesis .............................. SUCCESS [ 13.109 s]
[INFO] hazelcast-jet-s3 ................................... SUCCESS [01:16 min]
[INFO] hazelcast-jet-grpc ................................. SUCCESS [ 28.405 s]
[INFO] hazelcast-jet-protobuf ............................. SUCCESS [ 16.043 s]
[INFO] hazelcast-jet-python ............................... SUCCESS [ 51.435 s]
[INFO] hazelcast-distribution ............................. SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
```

Breaking test in `hazelcast-spring-tests`, breaks on Java 11 and Java 8

```
hazelcast-spring-tests % mvn test -Dtest=TestManagedContext
```

gives the following, not sure if mention of port 5702 is significant

```
[INFO]  T E S T S
[INFO] -------------------------------------------------------
[INFO] Running com.hazelcast.spring.context.TestManagedContext
    

[ERROR] Tests run: 10, Failures: 1, Errors: 5, Skipped: 0, Time elapsed: 235.488 s <<< FAILURE! - in com.hazelcast.spring.context.TestManagedContext
[ERROR] testDistributedTask(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 0.017 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.executor.impl.operations.MemberCallableTaskOperation, service: hz:impl:executorService
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:121)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.resolveAndThrowIfException(InvocationFuture.java:100)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.get(AbstractInvocationFuture.java:609)
	at com.hazelcast.spi.impl.DelegatingCompletableFuture.get(DelegatingCompletableFuture.java:109)
	at com.hazelcast.spring.context.TestManagedContext.testDistributedTask(TestManagedContext.java:91)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.executor.impl.operations.MemberCallableTaskOperation, service: hz:impl:executorService
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.initInvocationTarget(Invocation.java:290)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:562)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke0(Invocation.java:540)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke(Invocation.java:240)
	at com.hazelcast.spi.impl.operationservice.impl.OperationServiceImpl.invokeOnTarget(OperationServiceImpl.java:365)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:299)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:288)
	at com.hazelcast.spring.context.TestManagedContext.testDistributedTask(TestManagedContext.java:89)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:279)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:112)
	... 21 more

[ERROR] testRunnableTask(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 0.004 s  <<< ERROR!
com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.executor.impl.operations.MemberCallableTaskOperation, service: hz:impl:executorService
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.initInvocationTarget(Invocation.java:290)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:562)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke0(Invocation.java:540)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke(Invocation.java:240)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationBuilderImpl.invoke(InvocationBuilderImpl.java:59)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:415)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:434)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:343)
	at com.hazelcast.spring.context.TestManagedContext.testRunnableTask(TestManagedContext.java:107)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)

[ERROR] testRunnableTask_withScheduledExecutor_onRemoteMember(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 87.677 s  <<< ERROR!
com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.scheduledexecutor.impl.operations.ScheduleTaskOperation, service: hz:impl:scheduledExecutorService
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.initInvocationTarget(Invocation.java:290)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:562)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.access$400(Invocation.java:100)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation$InvocationRetryTask.run(Invocation.java:804)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:279)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapRuntimeException(AbstractInvocationFuture.java:1968)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapOrPeel(AbstractInvocationFuture.java:1949)
	at com.hazelcast.spi.impl.AbstractInvocationFuture$ExceptionalResult.wrapForJoinInternal(AbstractInvocationFuture.java:1431)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.resolveAndThrowForJoinInternal(AbstractInvocationFuture.java:600)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.joinInternal(AbstractInvocationFuture.java:584)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.submitOnMemberSync(ScheduledExecutorServiceProxy.java:511)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.scheduleOnMembers(ScheduledExecutorServiceProxy.java:311)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.scheduleOnMembers(ScheduledExecutorServiceProxy.java:289)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.scheduleOnMember(ScheduledExecutorServiceProxy.java:165)
	at com.hazelcast.spring.context.TestManagedContext.testRunnableTask_withScheduledExecutor_onRemoteMember(TestManagedContext.java:136)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)

[ERROR] testCallableTask_withScheduledExecutor_onRemoteMember(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 87.631 s  <<< ERROR!
com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.scheduledexecutor.impl.operations.ScheduleTaskOperation, service: hz:impl:scheduledExecutorService
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.initInvocationTarget(Invocation.java:290)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:562)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.access$400(Invocation.java:100)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation$InvocationRetryTask.run(Invocation.java:804)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:279)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapRuntimeException(AbstractInvocationFuture.java:1968)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.wrapOrPeel(AbstractInvocationFuture.java:1949)
	at com.hazelcast.spi.impl.AbstractInvocationFuture$ExceptionalResult.wrapForJoinInternal(AbstractInvocationFuture.java:1431)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.resolveAndThrowForJoinInternal(AbstractInvocationFuture.java:600)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.joinInternal(AbstractInvocationFuture.java:584)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.submitOnMemberSync(ScheduledExecutorServiceProxy.java:511)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.scheduleOnMembers(ScheduledExecutorServiceProxy.java:311)
	at com.hazelcast.scheduledexecutor.impl.ScheduledExecutorServiceProxy.scheduleOnMember(ScheduledExecutorServiceProxy.java:178)
	at com.hazelcast.spring.context.TestManagedContext.testCallableTask_withScheduledExecutor_onRemoteMember(TestManagedContext.java:150)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)

[ERROR] testTransactionalRunnableTask(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 60.008 s  <<< FAILURE!
java.lang.AssertionError: transaction manager could not proxy the submitted task.
	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.assertTrue(Assert.java:42)
	at com.hazelcast.spring.context.TestManagedContext.testTransactionalRunnableTask(TestManagedContext.java:167)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)

[ERROR] testTransactionalTask(com.hazelcast.spring.context.TestManagedContext)  Time elapsed: 0.004 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.executor.impl.operations.MemberCallableTaskOperation, service: hz:impl:executorService
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:121)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.resolveAndThrowIfException(InvocationFuture.java:100)
	at com.hazelcast.spi.impl.AbstractInvocationFuture.get(AbstractInvocationFuture.java:609)
	at com.hazelcast.spi.impl.DelegatingCompletableFuture.get(DelegatingCompletableFuture.java:109)
	at com.hazelcast.spring.context.TestManagedContext.testTransactionalTask(TestManagedContext.java:98)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: com.hazelcast.spi.exception.TargetNotMemberException: Not Member! target: [127.0.0.1]:5702, partitionId: -1, operation: com.hazelcast.executor.impl.operations.MemberCallableTaskOperation, service: hz:impl:executorService
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.initInvocationTarget(Invocation.java:290)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.doInvoke(Invocation.java:562)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke0(Invocation.java:540)
	at com.hazelcast.spi.impl.operationservice.impl.Invocation.invoke(Invocation.java:240)
	at com.hazelcast.spi.impl.operationservice.impl.OperationServiceImpl.invokeOnTarget(OperationServiceImpl.java:365)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:299)
	at com.hazelcast.executor.impl.ExecutorServiceProxy.submitToMember(ExecutorServiceProxy.java:288)
	at com.hazelcast.spring.context.TestManagedContext.testTransactionalTask(TestManagedContext.java:96)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.springframework.test.context.junit4.statements.RunBeforeTestExecutionCallbacks.evaluate(RunBeforeTestExecutionCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestExecutionCallbacks.evaluate(RunAfterTestExecutionCallbacks.java:84)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:75)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:115)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:107)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:829)
	at ------ submitted from ------.()
	at com.hazelcast.internal.util.ExceptionUtil.cloneExceptionWithFixedAsyncStackTrace(ExceptionUtil.java:279)
	at com.hazelcast.spi.impl.operationservice.impl.InvocationFuture.returnOrThrowWithGetConventions(InvocationFuture.java:112)
	... 21 more

[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   TestManagedContext.testTransactionalRunnableTask:167 transaction manager could not proxy the submitted task.
[ERROR] Errors: 
[ERROR]   TestManagedContext.testCallableTask_withScheduledExecutor_onRemoteMember:150 » TargetNotMember
[ERROR]   TestManagedContext.testDistributedTask:91 » Execution com.hazelcast.spi.except...
[ERROR]   TestManagedContext.testRunnableTask:107 » TargetNotMember Not Member! target: ...
[ERROR]   TestManagedContext.testRunnableTask_withScheduledExecutor_onRemoteMember:136 » TargetNotMember
[ERROR]   TestManagedContext.testTransactionalTask:98 » Execution com.hazelcast.spi.exce...
[INFO] 
[ERROR] Tests run: 10, Failures: 1, Errors: 5, Skipped: 0
```

--- a/hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/context/managedContext-applicationContext-hazelcast.xml
+++ b/hazelcast-spring-tests/src/test/resources/com/hazelcast/spring/context/managedContext-applicationContext-hazelcast.xml
@@ -39,9 +39,13 @@
             <hz:spring-aware/>
             <hz:cluster-name>test-managed-context</hz:cluster-name>
             <hz:network port="5701">
-                <hz:interfaces enabled="true">
-                    <hz:interface>127.0.0.1</hz:interface>
-                </hz:interfaces>
+                <hz:join>
+                    <hz:multicast enabled="false"/>
+                    <hz:tcp-ip enabled="true">
+                        <hz:interface>127.0.0.1:5701</hz:interface>
+                        <hz:interface>127.0.0.1:5702</hz:interface>
+                    </hz:tcp-ip>
+                </hz:join>
             </hz:network>
         </hz:config>
     </hz:hazelcast>
@@ -51,9 +55,13 @@
             <hz:spring-aware/>
             <hz:cluster-name>test-managed-context</hz:cluster-name>
             <hz:network port="5702">
-                <hz:interfaces enabled="true">
-                    <hz:interface>127.0.0.1</hz:interface>
-                </hz:interfaces>
+                <hz:join>
+                    <hz:multicast enabled="false"/>
+                    <hz:tcp-ip enabled="true">
+                        <hz:interface>127.0.0.1:5701</hz:interface>
+                        <hz:interface>127.0.0.1:5702</hz:interface>
+                    </hz:tcp-ip>
+                </hz:join>
             </hz:network>
         </hz:config>
     </hz:hazelcast>

--- a/hazelcast-spring/src/main/java/com/hazelcast/spring/cache/HazelcastCache.java
+++ b/hazelcast-spring/src/main/java/com/hazelcast/spring/cache/HazelcastCache.java
@@ -30,7 +30,7 @@ import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /**
- * Sprint related {@link Cache} implementation for Hazelcast.
+ * Spring related {@link Cache} implementation for Hazelcast.
  */
 public class HazelcastCache implements Cache {
 

--- a/hazelcast-spring/src/main/java/com/hazelcast/spring/hibernate/RegionFactoryBeanDefinitionParser.java
+++ b/hazelcast-spring/src/main/java/com/hazelcast/spring/hibernate/RegionFactoryBeanDefinitionParser.java
@@ -44,13 +44,13 @@ public class RegionFactoryBeanDefinitionParser extends AbstractBeanDefinitionPar
 
     @Override
     protected AbstractBeanDefinition parseInternal(Element element, ParserContext parserContext) {
-         NamedNodeMap attributes = element.getAttributes();
+        NamedNodeMap attributes = element.getAttributes();
         String instanceRefName = "instance";
         String mode = "DISTRIBUTED";
         if (attributes != null) {
             for (int a = 0; a < attributes.getLength(); a++) {
-                 Node att = attributes.item(a);
-                 String name = att.getNodeName();
+                Node att = attributes.item(a);
+                String name = att.getNodeName();
                 if ("instance-ref".equals(name)) {
                     instanceRefName = att.getTextContent();
                 } else if ("mode".equals(name)) {

